infrastructure/src/main/java/org/corfudb/infrastructure/ManagementServer.java

```diff
@@ -79,34 +85,31 @@ public ManagementServer(ServerContext serverContext) {
         this.healingPolicy = serverContext.getHealingHandlerPolicy();
 
         // Creating a management agent.
-        this.managementAgent = new ManagementAgent(this::getCorfuRuntime, serverContext);
-        this.orchestrator = new Orchestrator(this::getCorfuRuntime, serverContext);
+        this.managementAgent = new ManagementAgent(corfuRuntime, serverContext);
+        this.orchestrator = new Orchestrator(corfuRuntime, serverContext);
     }
 
     /**
      * Returns a connected instance of the CorfuRuntime.
      *
      * @return A connected instance of runtime.
      */
-    public synchronized CorfuRuntime getCorfuRuntime() {
-
-        if (corfuRuntime == null) {
-            CorfuRuntime.CorfuRuntimeParameters params =
-                    serverContext.getDefaultRuntimeParameters();
-            corfuRuntime = CorfuRuntime.fromParameters(params);
-            // Runtime can be set up either using the layout or the bootstrapEndpoint address.
-            if (serverContext.getManagementLayout() != null) {
-                serverContext
-                        .getManagementLayout()
-                        .getLayoutServers()
-                        .forEach(ls -> corfuRuntime.addLayoutServer(ls));
-            } else {
-                corfuRuntime.addLayoutServer(getBootstrapEndpoint());
-            }
-            corfuRuntime.connect();
-            log.info("getCorfuRuntime: Corfu Runtime connected successfully");
+    private CorfuRuntime getNewCorfuRuntime() {
+        final CorfuRuntime.CorfuRuntimeParameters params =
+                serverContext.getDefaultRuntimeParameters();
+        final CorfuRuntime runtime = CorfuRuntime.fromParameters(params);
+        // Runtime can be set up either using the layout or the bootstrapEndpoint address.
+        if (serverContext.getManagementLayout() != null) {
+            serverContext
+                    .getManagementLayout()
+                    .getLayoutServers()
+                    .forEach(ls -> runtime.addLayoutServer(ls));
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Replace this lambda with a method reference. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1612)